Microsoft.Toolkit.Uwp.UI/ImageCache/ImageCache.cs

```diff
@@ -138,6 +140,67 @@ public static string GetCacheFileName(Uri uri)
         }
 
         /// <summary>
+        /// Assures that image is available in the cache
+        /// </summary>
+        /// <param name="uri">Uri of the image</param>
+        /// <param name="storeToMemoryCache">Indicates if image should be available also in memory cache</param>
+        /// <returns>void</returns>
+        public static async Task PrecacheAsync(Uri uri, bool storeToMemoryCache = false)
```

Comment:
It does not return the BitmapImage. The point is it does not create BitmapImage for performance reasons, it just downloads the file if it is not there or it is outdated. If true is passed for storeToMemoryCache than it just calls GetFromCacheAsync because than BitmapImage has to be created to be stored in memory cache. Hope it makes sense.